src/diagonal.works/b6/expression.go

```diff
@@ -228,15 +232,15 @@ func (l Literal) ToProto() (*pb.NodeProto, error) {
 	return l.AnyLiteral.ToProto()
 }
 
-func LiteralFromProto(node *pb.NodeProto) (Expression, error) {
-	var e Expression
```

Comment:
This was the crux of the bug. e is defined here, but immediately shadowed below.

Reply:
UGH tyyyyy

Reply:
u didnt submit this

Reply:
Sorry, I've been a bit preoccupied and forgot about it! Rebasing and submitting now.